### PR TITLE
Cross-mio updates - added METCTDs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Added missing METCTDs, also added MIO as needed, and fixed Manufacturer SN for 37-11123 (had an 'E' at the end so I deleted that)
METCTDs added for WHOI:
CGINS-METCTD-13472
CGINS-METCTD-13544
CGINS-METCTD-13546
METCTDs added for OSU:
CGINS-METCTD-13468
CGINS-METCTD-13469
CGINS-METCTD-13470
CGINS-METCTD-13471

Can verify units via attached filtered Asset Trkg Ss.

[WHOI_Asset_Tracking_METCTDs.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/10134294/WHOI_Asset_Tracking_METCTDs.xlsx)

### CODE REVIEW

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 1]
```diff
@@ -1608,37 +1608,44 @@ CGINS-METBPR-00263,A01264,Sensor,0,Bulk Meteorology Instrument Package: METBK BP
...
+CGINS-METCTD-11560,A00752,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-11560,,20140312,5816,,Woods Hole Oceanographic Institution
+CGINS-METCTD-11561,A00715,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-11561,,20140227,5816,,Woods Hole Oceanographic Institution
+CGINS-METCTD-12754,A01266,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-12754,,20141214,5816,,Woods Hole Oceanographic Institution
+CGINS-METCTD-12755,A01267,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-12755,,20141214,5816,,Woods Hole Oceanographic Institution
+CGINS-METCTD-12756,A01268,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-12756,,20141214,5816,,Woods Hole Oceanographic Institution
+CGINS-METCTD-12757,A01269,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-12757,,20141214,5816,Woods Hole Oceanographic Institution,
```
Comment (snwhite) [inline bulk/sensor_bulk_load-AssetRecord.csv:1637]: Last comma should be before WHOI

**Review by snwhite — CHANGES_REQUESTED** [review]
One comma in the wrong place.

**Review by cdobs — APPROVED** [review]
Looks good

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 2]
```diff
@@ -1608,37 +1608,44 @@ CGINS-METBPR-00263,A01264,Sensor,0,Bulk Meteorology Instrument Package: METBK BP
...
-CGINS-METCTD-12757,A01269,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-12757,,20141214,5816,,
-CGINS-METCTD-12758,A01270,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-12758,,20141214,5816,,
+CGINS-METCTD-08028,A00207,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37SM61212-8028,,20101122,5816,,Woods Hole Oceanographic Institution
+CGINS-METCTD-10137,A00190,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-10137,,20130131,5816,Missing from inventory,Woods Hole Oceanographic Institution
+CGINS-METCTD-10138,A00197,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-10138,,20121226,5816,,Woods Hole Oceanographic Institution
+CGINS-METCTD-10139,A00204,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-10139,,20130131,5816,Missing from inventory,Woods Hole Oceanographic Institution
```
Comment (snwhite) [inline bulk/sensor_bulk_load-AssetRecord.csv:1614]: For 10137 and 10139, change Comment from "Missing from inventory" to "Lost at sea on CP01CNSM-00001".

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 3]
```diff
@@ -1608,37 +1608,44 @@ CGINS-METBPR-00263,A01264,Sensor,0,Bulk Meteorology Instrument Package: METBK BP
...
+CGINS-METCTD-10905,A00499,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-10905,,20131101,5816,,Oregon State University
+CGINS-METCTD-10906,A00498,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-10906,,20131101,5816.32,,Oregon State University
+CGINS-METCTD-11000,A00120,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-11000,,20121226,5816,,Oregon State University
+CGINS-METCTD-11001,A00121,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-11001,,20150604,5816,,Woods Hole Oceanographic Institution
+CGINS-METCTD-11002,A00477,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-11002,,20131021,5816,,Woods Hole Oceanographic Institution
+CGINS-METCTD-11003,A00440,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-11003,,20130930,5816,,Woods Hole Oceanographic Institution
```
Comment (snwhite) [inline bulk/sensor_bulk_load-AssetRecord.csv:1622]: For 11003, add Comment "Lost at sea on GI01SUMO-00004".

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 4]
```diff
@@ -1608,37 +1608,44 @@ CGINS-METBPR-00263,A01264,Sensor,0,Bulk Meteorology Instrument Package: METBK BP
...
+CGINS-METCTD-13471,A01859,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-13471,,,,,Oregon State University
+CGINS-METCTD-13472,A01860,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-13472,,,,,Woods Hole Oceanographic Institution
 CGINS-METCTD-13542,A01861,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-13542,,,,,Woods Hole Oceanographic Institution
 CGINS-METCTD-13543,A01862,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-13543,,,,,Woods Hole Oceanographic Institution
+CGINS-METCTD-13544,A01863,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-13544,,,,,Woods Hole Oceanographic Institution
 CGINS-METCTD-13545,A01864,Sensor,0,Bulk Meteorology Instrument Package: METBK CTD Module,METBK CTD MODULE,Sea-Bird Electronics,SBE 37-SM,37-13545,,,,,Woods Hole Oceanographic Institution
```
Comment (snwhite) [inline bulk/sensor_bulk_load-AssetRecord.csv:1647]: For 13545, add Comment "Lost at sea on GI01SUMO-00004".

**Review by snwhite — CHANGES_REQUESTED** [review]
A couple of comment changes requested.

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 2 above)
Comment (rtravis-whoi) [reply bulk/sensor_bulk_load-AssetRecord.csv:1614]: Done

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 3 above)
Comment (rtravis-whoi) [reply bulk/sensor_bulk_load-AssetRecord.csv:1622]: done

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 4 above)
Comment (rtravis-whoi) [reply bulk/sensor_bulk_load-AssetRecord.csv:1647]: done

**Review by snwhite — APPROVED** [review]

**Pull request merged**